android/app/build.gradle

```diff
@@ -92,13 +92,13 @@ def enableSeparateBuildPerCPUArchitecture = false
 def enableProguardInReleaseBuilds = false
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion 26
+    buildToolsVersion '25.0.3'
```

Comment:
Can the same up to 26 versions raise? I do not know for sure, just wondering.

Reply:
oopsy, updated